### ISSUE TITLE
Add native support for dark mode in SVG images

### ISSUE THREAD

**Issue by mgifford** [uswds/uswds#4305]
## Dark mode

Does the USWDS comply with the POUR framework of WCAG if the user has their browser (or operating system) set to dark mode? Not yet, but hopefully, that will change soon. 

I added this PR after looking through some of the SVGs and thinking that there must be a better way.

https://github.com/uswds/uswds/pull/4304

A few other related issues came up when I was looking at these. More standardization would be useful.

Essentially I specified a fill for `prefers-color-scheme: dark` - it would be useful if this were tested to see it worked well with custom style sheets. 

```
  <style>
    path {
      fill: #005EA2;
   }
   @media (prefers-color-scheme: dark) {
     path {
        fill: #58b4ff;
      }
    }
  </style>
```

**Comment by mgifford** [comment]
Also see https://github.com/uswds/uswds/issues/3449 & https://github.com/uswds/uswds/issues/4131

**Comment by mgifford** [comment]
I've created a darkmode version for all of the svg files currently in /src/img

These have been validated and some have been made more consistent.  The sprite at the end will be its own PR as it is a bit of a different beast.

This should be more accessible, however, we really need to re-think how we deal with CSS & SVGs.  This is a start, but I suspect we are going to need to see where this is applied to the themes. I found some odd combinations or inconsistencies.

why are there "alt" versions? What should happen to "white" checkboxes in dark mode? These may require further tweaking, but they have been adjusted to use `currentColor` where it made sense. 

https://jsfiddle.net/mgifford/28w1fr4h/

**Comment by mgifford** [comment]
First attempt to approach this - https://github.com/uswds/uswds/pull/4304

Most recent PR - https://github.com/uswds/uswds/pull/4321

**Comment by thisisdano** [comment]
Dark mode support makes sense, but it's not something we can apply only to SVGs — it will need comprehensive support across the design system, and that's going to be a really big thing. I support this work, but not in isolation.
